### PR TITLE
Conda-fy bamtools collection (take 2)

### PR DESCRIPTION
**Pull request by gregvonkuster**

### CODE REVIEW

**Comment by gregvonkuster** [comment]
Can this one be merged?

**Comment by gregvonkuster** [comment]
ping @bgruening

**Comment by bgruening** [comment]
@gregvonkuster for some reason no test is triggered. No idea currently why.

File: tool_collections/bamtools/bamtools_split/bamtools-split.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
-<tool id="bamSplit" name="Split" version="0.0.2" force_history_refresh="True">
+<tool id="bamSplit" name="Split" version="0.0.3" force_history_refresh="True">
```
Comment (nsoranzo) [inline tool_collections/bamtools/bamtools_split/bamtools-split.xml]: You should probably remove ` force_history_refresh="True"` (the XSD linter is complaining about it).

**Comment by gregvonkuster** [comment]
Thanks @nsoranzo, done.

**Comment by nsoranzo** [comment]
It would be nice to fix the Planemo linter warnings while we are on this: https://travis-ci.org/galaxyproject/tools-devteam/jobs/193054044#L816 .

Also what about syncing the wrapper version with the bamtools version (i.e. `2.4.0`)?

**Comment by gregvonkuster** [comment]
@bgruening @nsoranzo Hopefully should all be fixed now.

**Comment by nsoranzo** [comment]
Thanks @gregvonkuster, fantastic work!

**Comment by gregvonkuster** [comment]
Thanks for the merge! ;)

**Pull request merged**